script.js

```diff
@@ -34,10 +47,23 @@ function submitForm(e) {
   } else {
     error.innerHTML = '';
   }
+
+  saveData();
 }
 
```

Comment:
- [ ] Hi, your `saveData` function is correct 👍. although it shouldn't be called inside the `submitForm` function. the data is required to be saved every time the input changes, that is every time that the user types a key on the `input` fields. the purpose of this is so that when a user types some text and doesn't submit the form yet;  if they were to refresh the page the text they already wrote would still be available after the page refresh.
- [ ] Please make sure you add an `eventListener` that will fire every time the user clicks a key in the input fields and save that information to location storage.